chapters/statements.tex

```diff
@@ -34,11 +34,11 @@ \subsection{An Algorithm in a Model}\label{execution-of-an-algorithm-in-a-model}
 \item
   A continuous-time variable is initialized with the value of its \lstinline!start!-attribute.
 \item
-  A discrete-time variable \lstinline!v! is initialized with \lstinline!pre(v)!.
+  A discrete-time variable \lstinline!v! in a non-initial algorithm is initialized with \lstinline!pre(v)!.
```

Comment:
Aren't we talking any algorithm when being part of the initialization problem?  To me, an _initial algorithm_ sounds lika an `initial algorithm`.  Hence:
```suggestion
  During transient analysis, a discrete-time variable \lstinline!v! in an algorithm is initialized with \lstinline!pre(v)!.
```